wiki/en/Running-a-Server.md

```diff
@@ -125,185 +144,174 @@ To register with one of the Directories built into the Jamulus Client, replace `
+##### `--directoryfile` 
+_Directories only:_ Remember registered Servers even if the Directory is restarted. See [the Directories guide](Directories) for further information.
 
-**Note:** You will need to save recordings to a path _outside_ of the jamulus home Directory, or remove `ProtectHome=true` from your systemd unit file (be aware that doing so is however a potential security risk).
+##### `-f or --listfilter` 
+_Directories only:_ Whitelist Servers registering on the Server. See [the Directories guide](Directories) for further information.
 
-For turning recording on or off (depending on the current state):
-
-~~~
- [Unit]
- Description=Toggle recording state of Jamulus Server
- Requisite=Jamulus-Server
+---
 
- [Service]
- Type=oneshot
- ExecStart=/bin/systemctl kill -s SIGUSR2 Jamulus-Server
-~~~
+### General Server options
+These options can be used regardless of which mode your server is running in (although some may not be relevant for Directories).
 
-For starting a new recording:
+##### `-d or --discononquit` 
+Normally, when a Server is stopped or restarted, any Clients that have not used their “Disconnect” buttons will re-establish connection when the Server comes back up again. Using this option forces Clients to manually re-establish their connections to the Server.
 
-~~~
- [Unit]
- Description=Start a new recording on Jamulus Server
- Requisite=Jamulus-Server
+##### `-F or --fastupdate` 
+Reduces latency if Clients connect with “Enable Small Network Buffers” option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.
 
- [Service]
- Type=oneshot
- ExecStart=/bin/systemctl kill -s SIGUSR1 Jamulus-Server
-~~~
+##### `-l or --log` 
+Enable logging, set path and file name
 
-_Note: The Jamulus service name in the `ExecStart` line needs to be the same as the `.service` file name you created when setting systemd to control your Jamulus Server. So in this example it would be `Jamulus-Server.service`_
+##### `-m or --htmlstatus` 
+Enable HTML status file, set path and file name
 
-Run `sudo systemctl daemon-reload` to register them for first use.
+##### `-P or --delaypan` 
+Start with delay panning enabled.This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal “volume” panning.
 
-Now you can run these with the `systemctl` command, for example:
+##### `-s` or `--server`
+Start Jamulus in Server mode
 
-`sudo systemctl start jamulusTogglerec` (assuming you named your unit file `jamulusTogglerec.service`)
+##### `--serverbindip` 
+Specify the IP address the Jamulus process will bind to
 
-You can see the result of these commands if you run `systemctl status jamulus`, or by viewing the logs.
+##### `-T or --multithreading` 
+Use multithreading to make better use of multi-core CPUs to support more Clients
```

Comment:
Mm.  Probably want nice puntuation throughout the manual, rather than the brief form in the software...

Reply:
Not sure I understand. What do you mean by "brief form"?

Reply:
The form used in `--help`.  The basis of this file was just to list the content of the server `--help` output with almost no additional content.  That's pretty useless on its own.  As a _bare minimum_, I think this file should have correct language semantics (i.e. sentence structure and punctuation) where, perhaps, the `--help` output has aimed to be a "reminder".

Reply:
OK if there's stylistic stuff in here that you want to change, then feel free to suggest that. I wasn't planning that in this PR but no matter as I think most of it has been changed now!

Reply:
BTW don't forget we have this (which is made up of the include files) https://jamulus.io/wiki/Command-Line-Options because we used to have a standalone file for them originally and wanted to keep it for search engines. Dunno if that's useful/relevant in the context of this PR though.

Reply:
It's relevant in as much as its content shouldn't be repeated.  Saying "Use multithreading to make better use of multi-core CPUs to support more Clients" isn't really explanatory or useful to a server administrator -- it doesn't help them
- find out if it applies to them - i.e. how to determine how many cores they have
- judge the pros and cons of usage
- understand the threading model used in Jamulus

All of those would need explaining in a "Server Admin Guide".  Like I say, as this is _just_ a "re-structure", I'm not asking for all that to be done _now_.  What I am asking for is that corrections to sentence structure. including correct punctuation, are done to make the page look more "reviewed and professional" rather than just something "chucked together".

Reply:
This PR links the server options on that page to the admin guide, so it's not repeated.

And yes (and as per my summary notes) I am in this guide combining the options with the context of use and "supporting information" such as we have it right now (which sometimes we don't have enough as per your example). 

I think if you look at the changes I've committed from @mcfnord in this PR though, a lot of the suggestions he's made have been more "reviewed and professional".

Reply:
I consider the first sentence of a reference description to be nearly holy. It should summarize the feature effectively but concisely, with punctuation a close second concern. If a reference has a poor first sentence, the whole section is broken. If a reference _only_ has one accurate, adequate sentence, that could be sufficient for a V1.

Reply:
>  It should summarize the feature effectively but concisely

I consider this what the `--help` output is for.  The manual should provide additional details.  As it stands, the example here neither summarises the feature effectively - that would be achieved by something like "allows the use of multiple threads, where available, when mixing client audio" - nor is it properly punctuated.  (However, that suggestion also changes content, which is out of scope.)

Reply:
No probs - about 50 of its 53 commits on this PR are wording related, so I think we can change this :-)